backend/endpoints/sync.py

```diff
@@ -1,3 +1,44 @@
-from fastapi import APIRouter
+from bson.objectid import ObjectId
+from fastapi import APIRouter, HTTPException, status
+from fastapi.responses import JSONResponse
+from schema.response import ResponseModel
+from schema.room import RoomType
+from utils.sidebar import sidebar
 
 router = APIRouter()
+
+
+@router.get("/sidebar")
+async def get_sidebar(org_id: str, member_id: str, room_type: RoomType):
```

Comment:
```suggestion
async def get_sidebar(org_id: ObjectId, member_id: str, room_type: RoomType):
```
`org_id` parameter can now be of type `ObjectId`, right?

Reply:
Tried this, the server will not run because it complains of ObjectID not being a pyndantic data type

Reply:
Oh, ok